contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -114,7 +114,7 @@ BEGIN
         RAISE most_specific_type_mismatch;
 
     -- Check if arguments are out of range
-    ELSIF ((p_year NOT BETWEEN 1753 AND 9999) OR
+    ELSIF ((p_year NOT BETWEEN 0001 AND 9999) OR
```

Comment:
we only do changes in latest upgrade file which is 3.3 to 3.4, should not change 2.2 to 2.3 here

Reply:
Addressed